pages/common/sass.md

```diff
@@ -0,0 +1,20 @@
+# Sass
+
+> Converts SCSS or Sass files to CSS
+> .scss and .sass can be used interchangeably in below examples depending on your preference
+
+- Immediately convert scss file and output css with same filename
+
+`sass {{inputfile}}.scss`
+
+- Immediately convert SCSS or Sass file to CSS to specified output file
+
+`sass {{inputfile}}.scss {{outputfile}}.css`
+
+- Watch SCSS or Sass file for changes and output or update CSS file with same filename
+
+`sass --watch {{inputfile}}.scss`
+
+- Watch SCSS or Sass file for changes and output or update CSS file with specified filename
+
+`sass --watch {{inputfile}}.scss:{{outputfile}}.css`
```

Comment:
No, --watch will not allow you to use a space when specifying an output file. The single sass command will let you use a space OR a colon to delimit the input and output file but the watch command only allows for the colon delimiter. I used colon in the first commit as I felt it kept some consistency between the commands but it is less documented and is does not seem to be the standard so reverted to space delimiter.

![image](https://cloud.githubusercontent.com/assets/5971758/12102724/9c936d18-b2f1-11e5-8a12-cb6fdb11b037.png)